src/service/data_products/heatmap.py

```diff
@@ -10,6 +10,7 @@
 import src.common.storage.collection_and_field_names as names
 from src.common.product_models.common_models import FIELD_MATCH_STATE, FIELD_SELECTION_STATE
 from src.common.product_models import heatmap_common_models as heatmap_models
+from src.loaders.common.loader_helper import revert_transformed_heatmap_row_cells
```

Comment:
service code shouldn't depend on loader code and vice versa. This function should either go directly in heatmap or in src.common. Maybe there should be a specific module just for the two transform functions?

Reply:
👍 moved both transform functions to `heatmap_common_models`.

Reply:
:+1: